docs/src/booklet/6-solvers.md

```diff
@@ -0,0 +1,21 @@
+# QUBO Solvers
+
+## Solvers, Annealers & Samplers
+`ToQUBO`'s main goal is to benefit from non-deterministic samplers, especially *Quantum Adiabatic* devices and other *Annealing* machines. A few `MOI`-compliant interfaces for annealers and samplers are bundled within `ToQUBO` via the `Anneal.jl` submodule and package prototype. Some of them are presented below.
+
+### Simulated Annealing
+Provided by D-Wave's open-source code libraries, this [Simulated Annealing](https://en.wikipedia.org/wiki/Simulated_annealing) engine implements some of the features and configuration you would find using the Quantum API. Its adoption is recommended for basic usage, tests, and during early research steps due to its simplicity and ease of use. It does not implement the most advanced Simulated Annealing algorithm available but performs fairly well on small instances. `Anneal.jl` exports this interface as `SimulatedAnnealer.Optimizer`.
+
+### Quantum Annealing
+Interfacing with [D-Wave](https://www.dwavesys.com/)'s quantum computers is one of the milestones we expect to achieve with this package. Like other proprietary optimization resources such as [Gurobi](https://gurobi.com) and [FICO® Xpress](https://www.fico.com/en/products/fico-xpress-solver), this requires licensing and extra steps are needed to get access to it. In a first moment, for those willing to get started, the *Simulated Annealing* optimizer might be enough.
+
+While in `JuMP`, run `using Anneal` and look for `QuantumAnnealer.Optimizer`.
```

Comment:
Can we add the lines of code here? What is the user supposed to see?

Reply:
Yes, it would be nice to have an example on each solver highlighting the main features & setings.

Reply:
Code suggestions were removed for now